### PR TITLE
chore: Bump version to 1.41.1

### PR DESCRIPTION
**Pull request by mariusandra**
## Description

Update chart for 1.41.0

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mariusandra** [comment]
After making this PR I also see https://github.com/PostHog/charts-clickhouse/pull/610 - not sure what should get merged.
Here I bumped the major version based on @tiina303 's [comment here](https://posthog.slack.com/archives/C0374DA782U/p1667316396423159?thread_ts=1667315034.067579&cid=C0374DA782U)

File: charts/posthog/Chart.yaml [hunk 1]
```diff
@@ -11,11 +11,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 28.0.0
```
Comment (guidoiaquinti) [inline charts/posthog/Chart.yaml:14]: I don't think we need to bump the major version if we don't have release notes and/or breaking changes

Comment (mariusandra) [reply charts/posthog/Chart.yaml:14]: It's coming here:
https://github.com/PostHog/posthog.com/pull/4562

Comment (mariusandra) [reply charts/posthog/Chart.yaml:14]: The "here" will link to the array "[here](https://github.com/PostHog/posthog.com/blob/f153fd92fda5dfb26617db44e40fc6ec43ef3c5b/contents/blog/the-posthog-array-1-41-0.md#1410-update-guide-for-self-hosted-users)"

**Review by ellie — APPROVED** [review]

File: charts/posthog/Chart.yaml (hunk 1 above)
Comment (tiina303) [reply charts/posthog/Chart.yaml:14]: https://github.com/PostHog/posthog.com/pull/4562 WIP here

**Pull request merged**